index.bs

```diff
@@ -1266,7 +1266,9 @@ To <dfn>process an attributionsrc response</dfn> given a [=suitable origin=]
                 [=create an attribution trigger=] with |triggerHeader|
                 |destinationSite|, |reportingOrigin|, |privateStateTokens|, and
                 |context|'s [=current wall time=].
-            1. If |trigger| is not null, [=trigger attribution=] with |trigger|.
+            1. If |trigger| is not null:
+                1. [=Maybe defer trigger attribution=] with |trigger|.
```

Comment:
We probably need to specify that this needs to be done asynchronously somehow. Otherwise, waiting here will block all other processing, right?